packages/ember-auto-import/tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "noUnusedParameters": true,
     "sourceMap": true,
     "experimentalDecorators": true,
-    "strict": true
+    "strict": true,
+    "skipLibCheck": true
```

Comment:
I had to add this to fix `tsc` choking on the new embroider supplied types:

```
../../node_modules/@embroider/core/src/wait-for-trees.d.ts:6:5 - error TS2416: Property 'build' in type 'WaitForTrees<NamedTrees>' is not assignable to the same property in base type 'Plugin'.
  Type '(detail: { changedNodes: boolean[]; } | undefined) => Promise<void>' is not assignable to type '() => void | Promise<void>'.
```

🤔